Readme.md

```diff
@@ -1,206 +1,210 @@
+executed as our expection, then probably you have found a bug.
 
-Now we can use boards to organize tasks and lists to organize statuses, let's
-use Kanban to manage product development workflow.
+If reporter is not coming from a technical background (i.e. he is not a
+developer), ask the product manager to report the bug for you. Bugs get
+reported to github issues with detailed reports attached. In bug reports,
+reporter should provide a detailed procedure to reproduce the bug.
 
+#### Fixing bugs
 
-Ideas
-----
+Depending on how critical the bug is and how difficult to fix it, product
+manager decides to fix it in either this cycle (aka. `hot fix`) or the next
+one. It is also product manager's job to assign bugs to developers.
 
-Given its difficulty, product development is never an unattainable procedure.
-Like other engineering problems, we can solve it by careful analysis and
-planning. There are different stages during a full cycle of product development,
-to make tiny ideas going live.
+#### Merging bug fixes
 
-![ideas-board](images/ideas-board.png)
+Once fixed, assignee usually puts his/her clean fix in a single commit, leaving
+a message about which bug this commit fixes and which Github issue this bug
+associates. Then assignee submits his bug fix as a `pull request`, merging from
+bug-fix branch to production branch. Assignee should always reference the bug
+issue in pull request description and then assign this pull request to proper
+developer, who either owning the repository, or working on it.
 
-Any improvement of a product starts as an idea. To begin with, go to `Ideas`
-board, create a card in `Inbox` list, write down your idea.
+Each pull request requires at least one `LGTM` (looks good to me) or `SGTM`
+(seems good to me) comment to pass and get merged.
 
-Ideas could be a few sentences describing a problem users are facing, a user
-complain, or your vision about this product. It could be general and
-unorganized. Just leave it and product team will discuss and evaluate ideas in
-`Inbox` on the next `product review` meeting.
 
+### Technical Issues
 
-Task Lifecycle
-----
+> Technical issues are not fully designed features.
 
-<!-- TODO: explain -->
+Technical issues are spec problems. If set of bahaviors are designed but what
+happened in reality is not included, then probably you have found a technical
+issue.
 
-+ task waitlist (next-up)
-+ work in progress
-+ functional testing (staging)
-+ going live
+Technical issues get reported to github issues with a report attached. Like bug
+reports, this report contains detailed information about how to reproduce the
+missing scenarios. Product manager is responsible for distinguishing bugs and
+technical issues and writing reports.
 
-Each stage above has its corresponding task-list in `Bugs` and `Enhancements`
-board.
+Technical issues applies same [fixing](#fixing-bugs) and
+[merging](#merging-bug-fixes) strategies with bugs.
```

Comment:
@scottoasis How about always putting the Github issues link into the card description? Because as you have said:

> Technical issues get reported to github issues with a report attached